listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -274,9 +270,9 @@ def process_one_user(user):
         except spotify.SpotifyAPIError:
             current_app.logger.error('Could not refresh user token from spotify', exc_info=True)
             raise
-        else:
-            if user is None:
-                current_app.logger.debug("%s has revoked spotify authorization", str(user))
+        except spotify.SpotifyInvalidGrantError:
+            current_app.logger.error("%s has revoked spotify authorization", str(user), exc_info=True)
```

Comment:
I'm not sure that we should log this as an error. Consider that an error should probably be actionable by us, and in this case we can't do anything.
Because we have sentry, we'll get an email every time this happens. If we want to use sentry as a logger here, I think it'd be better to use info or similar. Additionally, it'd be great to have this grouped as a single error message - we could use parameters to store the username on each event.

Reply:
Agreed.